modules/translations/pages/index.adoc

```diff
@@ -26,6 +92,7 @@ field_en.xml
 ----
 
 The `<asset-type>` indicates the type of element the labels in the file are associated with. The possible asset types can be found in the xref:#asset-types[relevant section below]. The language identifier used is the standard 2 letter code in lowercase.
+There is a file named `language-codes.xml` (ending with .xml), this xref:#language-codes-xml[xml file] could be problematic for applications which do not expect the presence of this file, it is recommended to first go through the xref:#index[index file] found in the folder and get the desired xref:#files[filenames].
```

Comment:
This should be indicated as a [NOTE] (see the note at the top of the file), and could be a bit clearer by indicating that language-codes.xml does not contain translations and has a different format.

Reply:
It seems there are two ways to do a note, one that is more inline and another that is like a big block, I used the way it was done on top of the file, but I am not sure if I should use those ==== ... ==== around the text as those are not always used.
In the adoc docs they call this an "Admonition block", but it is not clear why we sometimes use it or not. Maybe just to better highlight where the note starts and ends?